src/Extension/Evaluator.cs

```diff
@@ -23,5 +27,35 @@ public Evaluation EvaluateResult(KernelCommandResult result)
             }
             return new Evaluation { Passed = true };
         }
+
+        public void AddInputExecutionCriterion(int cellId, string code)
```

Comment:
```suggestion
        public void AddInputExecutionCriterion(int questionId, string code)
```

Reply:
Refactored. The word "cell" is gone from the repo.